### DISCUSSION TITLE
Area Parameter of thumbnails/satellite/{area} request

### DISCUSSION THREAD

**Discussion by drmliam** [weather-gov/api#445] [Q&A]
I have been trying to use the https://api.weather.gov/thumbnails/satellite/{area} endpoint, but I have not discovered the format of the area string that is required for this request. I have seen no examples, so with a brief explanation and description of the area parameter could you also include an example request? Thank you

**Comment by StephenClouse** [comment]
I would advise not. This endpoint is basically non-functional and will be removed in a future release.

If you're looking for actual satellite imagery, you can obtain it from [NESDIS](https://www.nesdis.noaa.gov/).

**Reply by drmliam** [reply]
Thank you for your quick reply Stephen! Taking this into consideration I will look at NESDIS

again, thank you for your help!
